spec/user_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'rails_helper'
+
+RSpec.describe User, type: model do
+  subject { User.new(name: 'Midelaar', photo: 'http:/randomImage.where', bio: 'A humble man who loved his rubies.') }
+
+  before { subject.save }
+
+  it 'Name should be present' do
+    subject.name = nil
+    expect(subject).to_not_be_valid
+  end
+
+  it 'Post counter should be greater than 50 characters' do
+    subject.posts_counter = -1
+    expect(subject).to_not_be_valid
+  end
```

Comment:
- [ ] Good job so far 👏 Please add tests for the custom method in the `User` model called `give_recent_post` as I have seen in your model. This is among the project requirement and I would suggest that you test it too 😃 👍